### PR TITLE
Communication.md

### LINKED ISSUE #2: communication
The communication plan needs to be updated to reflect the latest changes in the team schedule and channels. This involves:
- Adding new communication channels
- Updating the schedule for meetings
- Clarifying the roles of team members

### LINKED ISSUE #14: Update Communication Plan
Update the communication plan to include revised schedules and processes for issue tracking.

### PR DESCRIPTION
**Pull request by AseelOmer**
This PR adds a new communication plan, including an updated schedule and task details for team collaboration.

### CODE REVIEW

**Review by Sufi-to — DISMISSED** [review]

File: collaboration/communication.md [hunk 1]
```diff
@@ -1,54 +1,35 @@
...
-______________________________________________________________________
+# Communication Plan
 
 ## Communication Schedule
 
-| Day | How | The topic of discussion | | --- | :-: | ----------------------- |
-| | | |
+| Day       | How (Channel)     | Topic of Discussion                                                    |
+|-----------|-------------------|------------------------------------------------------------------------|
+| Eveyday   | Slack             | Quick updates on progress and blockers.                                |
```
Comment (omniaNS) [inline collaboration/communication.md]: Typo: 'Eveyday' should be corrected to 'Everyday'

**Review by omniaNS — CHANGES_REQUESTED** [review]
Correct the typo in line 7 of communication.md

**Review by CaesarGhazi — COMMENTED** [review]
Looks great, other than that typo that Omnia pointed out i don't see anything wrong with it. GREAT JOB!!!!

**Comment by Nada-saad635** [comment]
GREAT JOB!

**Review by omniaNS — APPROVED** [review]

**Review by Nada-saad635 — COMMENTED** [review]
seems nice

**Review by Nada-saad635 — COMMENTED** [review]
.

**Comment by heba3** [comment]
Good job!

**Review by heba3 — COMMENTED** [review]
Good job!

**Pull request merged**